### PR TITLE
Feature/blog june 1 2023

### PR DESCRIPTION
**Pull request by dsarkar**

### CODE REVIEW

File: blog/2023-06-01-danke/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+page-title: 'The Corona-Warn-App team says &#34;Thank You!&#34;'
+page-description: 'The Corona-Warn-App team says &#34;Thank You!&#34;'
+page-name: thank-you
+page-name_de: thank-you
+author: CWA-Team, 10:00 am
+layout: blog
+---
+
+Today marks the end of more than three years of development work on one of the most important digital projects in the pandemic: the Corona Warning App. Since its launch on June 16, 2020, the CWA has been downloaded more than 48 million times. More than nine million alerts have been triggered since then, and many millions of people have been notified of risk encounters with red and green tiles on their smartphones.
```
Comment (MikeMcC399) [inline blog/2023-06-01-danke/index.md:10]: This website has consistently referred to CWA as "Corona-Warn-App" not "Corona Warning App" in English. The same term as is used in the English version of the app stores.

Later on in the text "Corona-Warn-App" is used.

So I suggest to change it here too.

Comment (dsarkar) [reply blog/2023-06-01-danke/index.md:10]: many thanks!

**Comment by Ein-Tim** [comment]
Deployment is failing: https://github.com/corona-warn-app/cwa-website/actions/runs/5141832260/jobs/9254790685

**Comment by MikeMcC399** [comment]
It's the same problem as yesterday:

```
Run azure/webapps-deploy@v[2](https://github.com/corona-warn-app/cwa-website/actions/runs/5141832260/jobs/9254790685#step:9:2)
  with:
    app-name: cwa-website
    slot-name: staging
    publish-profile: ***
    package: public
Package deployment using ZIP Deploy initiated.
Error: Failed to deploy web package to App Service.
Error: Deployment Failed, Error: Failed to deploy web package to App Service.
Internal Server Error (CODE: [5](https://github.com/corona-warn-app/cwa-website/actions/runs/5141832260/jobs/9254790685#step:9:5)00)
App Service Application URL: https://cwa-website-staging.azurewebsites.net/
```

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 Yes, we are looking at it right now.

**Pull request merged**